### PR TITLE
Holding github token

### PR DESCRIPTION
**Pull request by kotp**
ref: #1028

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
+github_token=$NOT_KYTRINYX_TOKEN
```
Comment (ErikSchierboom) [inline .travis.yml:2]: I don't know much about Travis, but should this definition be part of an `env` key: https://docs.travis-ci.com/user/environment-variables/#defining-public-variables-in-travisyml

Comment (ErikSchierboom) [reply .travis.yml:2]: The configlet script uses uppercase:
```suggestion
GITHUB_TOKEN=$NOT_KYTRINYX_TOKEN
```

**Comment by kotp** [comment]
@kytrinyx can you check Travis, last run was about a month ago for exercism/ruby from what I can tell.

**Comment by kotp** [comment]
Closing, as we have moved away from Travis.

**Pull request closed without merging**